avocado/core/nrunner.py

```diff
@@ -550,11 +550,9 @@ def _get_env_variables(self):
         return avocado_test_env_variables
 
     def run(self):
-        env = None
+        env = dict(os.environ)
```

Comment:
Could you please describe better the problem you are solving in the commit message?